python/docs/source/reference/pyspark.pandas/index.rst

```diff
@@ -23,7 +23,7 @@ Pandas API on Spark
 This page gives an overview of all public pandas API on Spark.
 
 .. note::
-   pandas API on Spark follows the API specifications of latest pandas release.
```

Comment:
I believe we cannot always support the latest pandas version in some cases, especially when they introduce the breaking changes.